cli/inspect.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/spf13/cobra"
+)
+
+// InspectCommand is used to implement 'inspect' command.
+type InspectCommand struct {
+	baseCommand
+}
+
+// Init initializes InspectCommand command.
+func (p *InspectCommand) Init(c *Cli) {
+	p.cli = c
+	p.cmd = &cobra.Command{
+		Use:   "inspect [container]",
+		Short: "get the detailed information of container",
+		Args:  cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return p.run(args)
```

Comment:
Could we use p.runInspect(args)?
Since this will be consistent with other commands.
Thanks a lot.